homeassistant/components/devolo_home_network/__init__.py

```diff
@@ -98,7 +98,12 @@ async def async_update_firmware_available() -> UpdateFirmwareCheck:
         try:
             return await device.device.async_check_firmware_available()
         except DeviceUnavailable as err:
-            raise UpdateFailed(err) from err
+            raise UpdateFailed(
+                err,
```

Comment:
Why is `err` passed in here?

Reply:
In https://github.com/home-assistant/core/pull/105755#discussion_r1523670117 @emontnemery requested to pass a non-translated string, so I did here, too.

Reply:
That is no longer needed. This parameter can be removed.